docs/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Estimator.md

```diff
@@ -16,7 +16,7 @@ Class for interacting with Qiskit Runtime Estimator primitive service.
 
 Qiskit Runtime Estimator primitive service estimates expectation values of quantum circuits and observables.
 
-The [`run()`](qiskit_ibm_runtime.Estimator#run "qiskit_ibm_runtime.Estimator.run") can be used to submit circuits, observables, and parameters to the Estimator primitive.
+The [`run()`](#qiskit_ibm_runtime.Estimator.run "qiskit_ibm_runtime.Estimator.run") can be used to submit circuits, observables, and parameters to the Estimator primitive.
```

Comment:
This change is fine. The link still works. Although I'm not certain why it happened.

Reply:
are you sure this works? I just tried in prod and that syntax doesn't seem interchangeable

Reply:
Yeah, it still works because we create two IDs for the same headers:

<img width="413" alt="Screenshot 2023-10-23 at 12 40 06 PM" src="https://github.com/Qiskit/documentation/assets/14852634/9492ff57-e9a6-48bf-98bf-8eda9cc6543b">

The `id` is what allows the anchor to work.

This continues to be the case.

The other important detail is that the link is in the same file, which is why it's fine to have `#` in the beginning. Before, it was saying go to the file `qiskit_ibm_runtime.Estimator`, then use the anchor `#run`